automounterhelper 1.6.0 (new cask)

a-mnich opened the pull request:
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [X] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [X] `brew audit --cask --online <cask>` is error-free.
- [X] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [X] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [X] `brew audit --cask --new <cask>` worked successfully.
- [X] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [X] `brew uninstall --cask <cask>` worked successfully.

---

khipp commented on Casks/a/automounterhelper.rb:
```diff
@@ -0,0 +1,19 @@
+  homepage "https://pixeleyes.co.nz/automounter/helper/"
+
+  livecheck do
+    url "https://pixeleyes.co.nz/automounter/helper/"
+    regex(/version: (\d+.\d+.\d+)/i)
+    strategy :page_match
```
```suggestion
    url :homepage
    regex(/Version:\s+v?(\d+(?:\.\d+)+)/i)
```

khipp commented:
Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

a-mnich commented:
Hi @khipp, thanks for reviewing.
I've included your proposed changes and added `# No zap stanza required` as stated in the docs as I and `nrlquaker/createzap` didn't find any files to remove. I've also corrected the commit message according to the guidelines.

khipp commented:
Thank you for your first contribution to `homebrew-cask`, @a-mnich! 🎉
